test/groovy/util/StepTracker.groovy

```diff
@@ -0,0 +1,42 @@
+package util
+
+import groovy.json.JsonBuilder
+
+class StepTracker {
+
+        final static filePath = 'target/trackedCalls.json'
+
+    final static trackedCalls = [:]
+
+    def add(String testClass, Collection steps) {
+
+        def stepName = stepName(testClass)
+
+        if(trackedCalls[stepName] == null)
+            trackedCalls[stepName] = (Set)[]
+
+        steps.each { c -> trackedCalls[stepName] << c.name }
+
+        persist()
```

Comment:
In fact we persist ways to often ...
Would be sufficient to persist only once, after the last test. But since we do not know which test is the latest one, there is no other option than persisting each time ...